src/Forecast.js

```diff
@@ -0,0 +1,61 @@
+import { Grid, Box, Dialog, DialogTitle } from "@mui/material";
+import {
+  LineChart,
+  CartesianGrid,
+  XAxis,
+  YAxis,
+  Line,
+  Tooltip,
+} from "recharts";
+
+export default function Forecast(props) {
+  const { forecastData } = props;
+  if (!("city" in forecastData)) {
```

Comment:
Instead of making this a null rendering component, why not use this as conditional render logic in the first place for Forecast?